code/modules/mob/living/simple_animal/hostile/ordeal/white.dm

```diff
@@ -11,8 +11,9 @@
 	health = 3000
 	melee_damage_type = BLACK_DAMAGE
 	rapid_melee = 2
-	melee_damage_lower = 30
-	melee_damage_upper = 40
+	melee_damage_lower = 50
```

Comment:
i don't know if making this guy a more brutal straight up fight is... great for him overall? A lot of his danger comes from infinite breaching abnos, this change makes him a massive melee bruiser on top of that.

Reply:
I'll clarify here Im down with the hammer changes because the hammer is a fairly reactable part of the fight that lets you hit him in response to him using a powerful move, something that's being heavily reduced for Red Fixer in this same PR.

Reply:
He does not breach as much as he used long ago, and basing the entire ordeal around it is silly, unless we buff the breach ability again.